### ISSUE TITLE
Malfunction of some gestures

### ISSUE THREAD

**Issue by ichnilatis-gr** [koreader/koreader#7727]
* KOReader version: 2021/05
* Device: Pocketbook Touch Lux 5 (PB628)

After last update, some gestures do not work properly.
When I make the gesture in a pdf to zoom it to contentwidth, pages are displayed one inside the other, as below:

![Reader_2021-05-20_080322_LI](https://user-images.githubusercontent.com/76203123/119137046-0b4e6c80-ba49-11eb-82ed-ef2a157b4fb8.jpg)

![Reader_2021-05-20_213801_LI (2)](https://user-images.githubusercontent.com/76203123/119137321-5d8f8d80-ba49-11eb-8f61-9c57bef0f8ec.jpg)

Also, when I make the spread gesture in order to increase the letters in an epub, they decrease. In fact either spread or pinch gestures do not function well.

Thank you for your help in advance!

**Comment by NiLuJe** [comment]
> When I make the gesture in a pdf to zoom it to contentwidth, pages are displayed one inside the other, as below:

Which gesture would that be?

----

Also, since this is PB, and input on PB is a special snowflake: Is the device rooted?

----

In any case: verbose debug logs, please :).

**Comment by ichnilatis-gr** [comment]
> Which gesture would that be?

It's about the multi-gesture →↑←→ 
In fact, the problem occurred in a djvu file (from which the icons above)

Also, there is a problem with the gesture I've set for file browser (↑↓), it closes KOReder and it goes me to Pocketbook main menu. The same happens with the file browser button at the upper menu. It closes KOReader too.

>  Is the device rooted?

I'm sorry. I don't understand the question. What do you mean exactly?

> In any case: verbose debug logs, please :).

Could you explain to me how I could do this?

Thank you!

**Comment by NiLuJe** [comment]
Tools > More tools > Developer options

Tick both debug logging & verbose logging, restart KOReader.

Try everything that doesn't work/crash, and then post the `crash.log` file (in koreader's folder).

I'd update to the latest [nightly](http://build.koreader.rocks/download/nightly/) first, too.

**Comment by NiLuJe** [comment]
Also, more details about your PDF setup would help, because I can't reproduce any of this ;).

(I mean besides pinch/spread gestures being super-finicky, but that's nothing new).

**Comment by ichnilatis-gr** [comment]
Dear NiLuJe, 
It seems that the problems have been sold.

I installed the last nightly version I had before 2021/05 (i.e. koreader-pocketbook-v2021.04-79-gba00f38_2021-05-16) and I saw that the problems continued to exist. I deleted two plugins I had installed and I installed again version 2021/05. The problem with the file browser gesture and button was solved.

Also I noticed that the problem with the zoom to widthcontent gesture only appeared in this particular file.

Thank you for all!
